docker/Dockerfile

```diff
@@ -16,7 +16,7 @@
 FROM python:3.7-slim
 WORKDIR /root
 
-FROM pytorch/pytorch:1.6.0-cuda10.1-cudnn7-runtime
+FROM pytorch/pytorch:1.4-cuda10.1-cudnn7-runtime
```

Comment:
Yeah, for the current setup of NNI, pytorch:1.4 is needed. It won't affect adaptdl-texar in my test but I am not sure if it will affect other examples.